dataproc/src/test/java/CreateClusterTest.java

```diff
@@ -38,18 +38,18 @@
 @RunWith(JUnit4.class)
 public class CreateClusterTest {
 
-  private static final String BASE_CLUSTER_NAME = "test-cluster";
+  private static final String CLUSTER_NAME =
+      String.format("java-dataproc-create-cluster-test-%s", UUID.randomUUID().toString());
   private static final String REGION = "us-central1";
+  private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
 
-  private static String projectId = System.getenv("GOOGLE_CLOUD_PROJECT");
-  private String clusterName;
   private ByteArrayOutputStream bout;
+  private PrintStream standardOutOrig;
 
   private static void requireEnv(String varName) {
     assertNotNull(
```

Comment:
Discovered in another PR we've been doing this backwards - it needs to be `assertNotNull(msg, value)` to work correctly.